qemu/tests/cfg/multi_nics_hotplug.cfg

```diff
@@ -26,9 +26,8 @@
             nics = ""
             extra_params += "-net none"
             s390x:
-            ''' virtio-net-ccw is not a kind of pci device, but
-            more convinient to use pci_model here
-            '''
+            # virtio-net-ccw is not a kind of pci device, but more
+            # convenient to use pci_model here
```

Comment:
It is better to align with `pci_model = virtio-net-ccw` , or move to the front of `s390x:`.

Reply:
updated